src/main/kotlin/no/nav/sbl/dsop/api/Bootstrap.kt

```diff
@@ -27,13 +31,31 @@ fun main(args: Array<String>) {
     start(webApplication())
 }
 
-fun webApplication(port: Int = 8080, mockdata: Any? = null): ApplicationEngine {
+fun webApplication(port: Int = 8080, mockdata: Any? = null, env: Environment = Environment()): ApplicationEngine {
     return embeddedServer(Netty, port) {
-        install(StatusPages) {
+         install(StatusPages) {
             status(HttpStatusCode.NotFound) { cause ->
                 KLogging().logger.warn(cause.description + ": " + call.request.uri)
             }
         }
+
+        val conf = this.environment.config
+        (conf as MapApplicationConfig).apply {
+            put("no.nav.security.jwt.issuers.size", "1")
+            put("no.nav.security.jwt.issuers.0.issuer_name", env.securityJwksIssuer)
+            put("no.nav.security.jwt.issuers.0.discoveryurl", env.securityJwksUrl)
+            put("no.nav.security.jwt.issuers.0.accepted_audience", env.securityAudience)
+            put("no.nav.security.jwt.issuers.0.cookie_name", OIDC_COOKIE_NAME)
+        }
```

Comment:
Her er artikkelen jeg nevnte for å kunne sette properties via Ktor sin properties-fil: https://ktor.io/quickstart/quickstart/gradle.html